classes/notification/NotificationAPIManager.php

```diff
@@ -356,7 +356,43 @@ public function getUserNotifications(): array
                 alert: false,
             );
         }
-
+        //Caravan
+        $time = time();
+        $result = $this->system->db->query("SELECT `caravans`.*, `regions`.`name` as 'region_name' FROM `caravans` INNER JOIN `regions` on `caravans`.`region_id` = `regions`.`region_id` WHERE `start_time` < {$time}");
+        $result = $this->system->db->fetch_all($result);
+        foreach ($result as $row) {
+            // if travel time is set then only display if active
+            if (!empty($row['travel_time'])) {
+                if ($row['travel_time'] + ($row['start_time'] * 1000) + Patrol::DESTINATION_BUFFER_MS > (time() * 1000)) {
```

Comment:
It would be a persistent notification for all currently active caravans. 
15 spread across 6 hours with 5 min travel that'll usually be 0 or 1, occasionally 2.